packages/@o3r/application/schematics/ng-add/index.ts

```diff
@@ -1,6 +1,60 @@
+import {chain} from '@angular-devkit/schematics';
 import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import {getAppModuleFilePath, insertImportToModuleFile, addImportToModuleFile as o3rAddImportToModuleFile} from '@o3r/schematics';
```

Comment:
should be a dynamic import to handle the case this `ng add` is executed in the app without O3r Core (as done in the other ng-add schematics.